models/dao/dao_schema.yml

```diff
@@ -0,0 +1,95 @@
+version: 2
+
+models:
+  - name: dao_votes
+    meta:
+      blockchain: ethereum
+      sector: dao
+      contributors: soispoke
+    config:
+      tags: ['ethereum', 'votes', 'dao', 'cross-chain']
+    description: >
+      DAO votes on all chains across all contracts and versions
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain which the DAO is deployed"
+      - &project
+        name: project
+        description: "Project name of the DAO"
+      - &version
+        name: version
+        description: "Version of the contract built and deployed by the DAO project"
+      - &block_time
+        name: block_time
+        description: "UTC event block time of each DAO vote"
+      - &tx_hash
+        name: tx_hash
+        description: "Unique transaction hash value tied to each transaction on the DEX"
+      - &dao_name
+        name: dao_name
+        description: "DAO name"
+      - &dao_address
+        name: dao_address
+        description: "DAO wallet address"
+      - &proposal_id
+        name: proposal_id
+      - &votes
+        name: votes
+        description: "Votes weighted by the amount of governance tokens"
+      - &votes_share
+        name: votes_share
+        description: "Votes share in percent for a given proposal"
+      - &token_symbol
+        name: token_symbol
+      - &token_address
+        name: token_address
+      - &votes_value_usd
+        name: votes_value_usd
+        description: "Value in USD corresponding to the governance tokens used to vote on a given proposal"
+      - &voter_address
+        name: voter_address
+      - &support
+        name: support
+        description: "For, Against or Abstain votes"
+      - &reason
+        name: reason
+        description: "Optional onchain commments to explain votes"
+
+
+  - name: dao_proposals
+    meta:
+      blockchain: ethereum
+      sector: dao
+      contributors: soispoke
+    config:
+      tags: ['ethereum', 'proposals', 'dao', 'cross-chain']
+    description: >
+      DAO proposals on all chains across all contracts and versions
+    columns:
+      - *blockchain
+      - *project
+      - *version
+      - *block_time
```

Comment:
For votes it seems fairly obvious that block_time = time the vote was cast
Probably here it means time when the proposal was created? Or does it mean this is the time as-of-when the vote counts are the ones below? Maybe we should rename to clarify?

Reply:
Yes, agree it would make more sense to have the column renamed as "created_at" (open to other suggestions if you don't like it) for proposals 👍 Done